tests/auth_tests/test_validators.py

```diff
@@ -13,6 +13,8 @@
     validate_password,
 )
 from django.core.exceptions import ValidationError
+from django.db.models import Model
```

Comment:
It's more common to use `from django.db import models` and refer to `models.Model` and `models.EmailField`.